bigbluebutton-html5/imports/startup/client/auth.js

```diff
@@ -29,43 +33,43 @@ export function joinRouteHandler(nextState, replace, callback) {
         externUserID, fullname, confname, customdata,
       } = response;
 
+      console.log({ returncode });
       if (returncode === 'FAILED') {
-        replace({ pathname: '/error/404' });
-        callback();
+        Session.set('hasError', true);
+        Session.set('codeError', '404');
+        callback('failed unhappily');
```

Comment:
the expected params are `(value, error)` you probably should change the callback call or the parameters to reflect the return.